code/game/objects/items/weapons/melee/energy_melee_weapons.dm

```diff
@@ -315,7 +322,7 @@
 		return
 	var/datum/status_effect/saw_bleed/B = target.has_status_effect(STATUS_EFFECT_SAWBLEED)
 	if(!B)
-		if(!active) //This isn't in the above if-check so that the else doesn't care about active
+		if(HAS_TRAIT(src, TRAIT_ITEM_ACTIVE)) //This isn't in the above if-check so that the else doesn't care about active
```

Comment:
```suggestion
		if(!HAS_TRAIT(src, TRAIT_ITEM_ACTIVE)) //This isn't in the above if-check so that the else doesn't care about active
```
surely we should be checking if it is inactive? like the old code used to?